docs/src/pages/component-demos/toggle-button/NormalToggleButton.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, createStyleSheet } from 'material-ui/styles';
+import ToggleButton, { Option } from 'material-ui/ToggleButton';
+import Icon from 'material-ui/Icon';
+import { MenuItem } from 'material-ui/Menu';
+import Paper from 'material-ui/Paper';
+import Typography from 'material-ui/Typography';
+
+const styleSheet = createStyleSheet('NormalToggleButton', {
+  root: {
+    width: '100%',
+  },
+  paper: {
+    padding: 30,
+  },
+});
+
+function NormalToggleButton(props) {
+  const classes = props.classes;
+
+  function changeColor(value) {
+    const div = document.getElementById('dummyDiv').style;
+    if (value === 4) {
+      div.background = '#F44336';
+    } else if (value === 5) {
+      div.background = '#2196F3';
+    } else if (value === 6) {
+      div.background = '#009688';
+    }
+  }
+
+  function colorReset() {
+    document.getElementById('dummyDiv').style.background = 'white';
+  }
+
+  function changeText(value) {
+    const text = document.getElementById('dummyText').style;
+    if (value === 1) {
+      text.fontWeight = 800;
+    } else if (value === 2) {
+      text.fontStyle = 'italic';
+    } else if (value === 3) {
+      text.textDecoration = 'underline';
+    }
+  }
+
+  function resetText(value) {
+    const text = document.getElementById('dummyText').style;
+    if (value === 1) {
+      text.fontWeight = 'normal';
+    } else if (value === 2) {
+      text.fontStyle = 'normal';
+    } else if (value === 3) {
+      text.textDecoration = 'none';
+    }
+  }
+
+  return (
+    <div className={classes.root}>
+      <ToggleButton>
+        <Option
+          icon={<Icon className="material-icons">format_bold</Icon>}
+          value={1}
+          onSelect={changeText}
+          onDeselect={resetText}
+        />
+        <Option
+          icon={<Icon className="material-icons">format_italic</Icon>}
+          value={2}
+          onSelect={changeText}
+          onDeselect={resetText}
+        />
+        <Option
+          icon={<Icon className="material-icons">format_underline</Icon>}
+          value={3}
+          onSelect={changeText}
+          onDeselect={resetText}
+        />
+        <Option
+          icon={<Icon className="material-icons">format_color_fill</Icon>}
+          onSelect={changeColor}
+          onDeselect={colorReset}
+        >
+          <MenuItem value={4}>Red</MenuItem>
+          <MenuItem value={5}>Blue</MenuItem>
+          <MenuItem value={6}>Green</MenuItem>
+        </Option>
+      </ToggleButton>
```

Comment:
I don't think this should be part of the demo because it doesn't seem to be in the scope of a toggle button's user experience.  I think this is more like a dropdown menu.  Hopefully we can open a discussion on this.